File: python/1010.py
# -*- coding: utf-8 -*-

a, b, c = input().split(" ")
a = int(a)
b = int(b)
c = float(c)
# b = int(input(""))
# c = float(input(""))
d, e, f = input().split(" ")
d = int(d)
e = int(e)
f = float(f)
# e = int(input(""))
# f = float(input(""))
tot = (b * c) + (e * f)

print("VALOR A PAGAR: R$ %.2f" %tot)

File: C/1010.c
#include <stdio.h>

int main(){
    int a, b, d, e;
    float c, tot = 0, f, tot1 = 0;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%f", &c);
    tot = b * c;
    scanf("%d", &d);
    scanf("%d", &e);
    scanf("%f", &f);
    tot1 = e * f;
    printf("VALOR A PAGAR: R$ %.2f\n", (tot + tot1));
    return 0;
}
